package.json

```diff
@@ -25,21 +26,23 @@
     "url": "https://github.com/alexanderzobnin/grafana-zabbix/issues"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-react": "^7.6.3",
-    "@grafana/data": "^6.4.3",
-    "@grafana/runtime": "^6.4.3",
-    "@grafana/toolkit": "^6.4.3",
-    "@grafana/ui": "^6.4.3",
+    "@babel/core": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@babel/preset-react": "^7.8.3",
+    "@emotion/core": "^10.0.27",
+    "@grafana/data": "canary",
+    "@grafana/runtime": "canary",
+    "@grafana/toolkit": "canary",
+    "@grafana/ui": "canary",
     "@types/classnames": "^2.2.9",
     "@types/grafana": "github:CorpGlory/types-grafana",
     "@types/jest": "^23.1.1",
     "@types/jquery": "^3.3.0",
     "@types/lodash": "^4.14.104",
     "@types/moment": "^2.13.0",
-    "@types/react": "^16.4.6",
-    "@types/react-dom": "^16.0.11",
+    "@types/react": "^16.9.17",
+    "@types/react-table": "^6.8.6",
```

Comment:
Current builds of @grafana/ui started using react-table 7. Without defining our own version of the types here, it was using types 7.x and throwing errors due to type changes in the next version.